README.md

```diff
@@ -55,14 +55,23 @@ If you would like to consider contributing to the project, please see our [Contr
 * Open a [GitHub issue](../../issues) to provide feedback, raise issues and ask questions.
 * Report a security vulnerability according to the [Reporting Vulnerabilities guide](https://www.oracle.com/corporate/security-practices/assurance/vulnerability/reporting.html). More information at [SECURITY](SECURITY.md).
 
-## License
-
-`Agda-LBFT` is [licensed](LICENSE.txt) under [UPL 1.0](https://opensource.oracle.com/licenses/upl).
+## Contributing
 
-## Contributors
+This project welcomes contributions from the community. Before submitting a pull request, please [review our contribution guide](./CONTRIBUTING.md)
 
 As of the beginning of this repo, contributions have been made by
 [Victor Cacciari Miraldo](https://github.com/VictorCMiraldo), [Harold Carr](https://github.com/haroldcarr), [Mark Moir](https://github.com/mark-moir), and [Lisandra Silva](https://github.com/lisandrasilva), all while employed at Oracle Labs.
 
 We are grateful to the following people for contributions since then:
 * [Chris Jenkins](https://github.com/cwjnkins)
+
+## Security
+
+Please consult the [security guide](./SECURITY.md) for our responsible security vulnerability disclosure process
+
+## License
+
+Copyright (c) 2020, 2023 Oracle and/or its affiliates.
```

Comment:
Should this be 2020-2023, or maybe 2020, 2021, 2022, 2023, or is it OK as is?

Reply:
according to https://github.com/oracle/template-repo/blob/main/README.md:
The correct copyright notice format for both documentation and software is "Copyright (c) [year,] year Oracle and/or its affiliates."
You must include the year the content was first released (on any platform) and the most recent year in which it was revised